net/wavemon/Makefile

```diff
@@ -28,7 +28,7 @@ define Package/wavemon
   SECTION:=net
   CATEGORY:=Network
   TITLE:=N-curses based wireless network devices monitor
-  DEPENDS:=+libncurses +libpthread +libnl-genl
+  DEPENDS:=+libncurses +libpthread +libnl-genl +libnl-route +libnl-nf +libnl-cli
```

Comment:
So many dependencies were added and there wasn't anything about this in the commit description